exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,6 @@
-// Declare your function first
-
-var result = divide(3, 4);
-
-console.log(result);
+function divide(num1, num2) {
```

Comment:
it's very helpful if you add comment before function to explain what is function about . :)